assets/js/setup.js

```diff
@@ -293,6 +295,26 @@ function customFunctionOnMove() {
 
     // Update autoplay button (e.g. in case of clicking on move in PGN window)
     setAutoplayButton();
+
+    // start clock countdown if enabled
+    if (clockCountdownEnabled) {
+        // get reference time from pgn
+        referenceTime = customPgnHeaderTag("ReferenceTime");
+
+        // remove clockActive if set from previous game
+        document.getElementById("ClockPlace1").classList.remove("clockActive");
+        document.getElementById("ClockPlace2").classList.remove("clockActive");
```

Comment:
It seems more logical to do this in `customFunctionOnPgnGameLoad()`.

I tried it quickly, but it seems there is a significant delay before re-drawing the colored timebox (when switching between games). This can be checked sometimes later...